.java-version

```diff
@@ -0,0 +1 @@
+1.8
```

Comment:
Oh, missed this comment. I use this for `jenv`, but if someone minds it I'm happy to ignore it from git commits.